Client/src/App.js

```diff
@@ -1,11 +1,17 @@
+/** [App.js]
+* Summary.
+AuthProvider wraps the Routes so that we can check when loggedIn is true.
```

Comment:
Uncertain if app.js has a differing purpose in front-end. @MichaelDCristiano can you confirm?

Reply:
Confirmation from Michael has been provided.